bin/migrate

```diff
@@ -3,4 +3,4 @@ set -e
 
 python manage.py migrate
 python manage.py migrate_clickhouse
-python manage.py run_async_migrations
\ No newline at end of file
+python manage.py run_async_migrations --check
```

Comment:
This breaks fresh installs. The reason we have this here is so that on a fresh install we just skip and mark as complete the non-required migrations.

Reply:
We should be completing all migrations that can be completed from `--check` too, see https://github.com/PostHog/posthog/blob/master/ee/management/commands/run_async_migrations.py#L39